### PR TITLE
Make textbox draggable when text is edting

### PR DESCRIPTION
**Pull request by lpranam**
* Target version: master 

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: browser/src/layer/vector/SVGGroup.js [hunk 1]
```diff
@@ -165,7 +165,7 @@ L.SVGGroup = L.Layer.extend({
 	_onDragStart: function(evt) {
 		if (!this._map || !this._dragShapePresent || !this.dragging)
 			return;
-		if (this._map._docLayer._cursorMarker && this._map._docLayer._cursorMarker.isVisible())
+		if (this._map._docLayer._cursorMarker && $('.leaflet-pane.leaflet-map-pane').css('cursor') === 'text')
```
Comment (eszkadev) [inline browser/src/layer/vector/SVGGroup.js:168]: This query can fail and cause error.
Also please don't use queries when possible. Can't we get object by getting the section reference and using it's member?

**Comment by eszkadev** [comment]
My impression is that cursor change was always very slow/delayed. Is that reliable approach to rely on that?

**Comment by eszkadev** [comment]
I see that in the core we have 2 modes too: editing text, just shape selection:
selection

It is possible to drag if user has a pointer on a border. Could it be possible to detect here the border too? And then allow dragging.

**Comment by lpranam** [comment]
But the thing is in core you can drag also when you are editing text(which had border) if you start dragging from part of the box which does not have text.

**Comment by eszkadev** [comment]
And that is my question: is that possible to do also in online? I guess we send wrong commands because we are in edit mode. But we could add some overlay element which will do different thing... and will move the shape by sending other command.

**Comment by lpranam** [comment]
close in favor of https://github.com/CollaboraOnline/online/pull/8419

**Pull request closed without merging**